app/webpack/stylesheets/components/_typography.scss

```diff
@@ -0,0 +1,6 @@
+.vcd-body-secondary {
```

Comment:
It would be ideal to move this file to a directory titled 'core', i.e. `app/webpack/stylesheets/core/_typography.scss`